docs/installation/config_cli.rst

```diff
@@ -5,216 +5,73 @@
-    ...
-
-.. note:: The ``uuid`` field will be used to lookup existing ``ObjectType``'s.
-
-Objecttypes require a corresponding ``Service`` to work correctly. Creating
-these ``Service``'s can be done by defining these in the same yaml file. ``Service``
-instances will be created before the ``ObjectType``'s are created.
-
-Objecttypes connection configuration
-------------------------------------
-
-In order to be able to retrieve objecttypes, a corresponding ``Service`` should be
-created. An example of a configuration could be seen below:
+.. setup-config-usage::
+  :show_command_usage: false
 
-.. code-block:: yaml
-
-    zgw_consumers_config_enable: true
-    zgw_consumers:
-      services:
-      - identifier: objecttypes-api-1
-        label: Objecttypes API 1
-        api_root: http://objecttypes-1.local/api/v1/
-        api_connection_check_path: objecttypes
-        api_type: orc
-        auth_type: api_key
-        header_key: Authorization
-        header_value: Token ba9d233e95e04c4a8a661a27daffe7c9bd019067
-      - identifier: objecttypes-api-2
-        label: Objecttypes API 2
-        api_root: http://objecttypes-2.local/api/v1/
-        api_connection_check_path: objecttypes
-        api_type: orc
-        auth_type: api_key
-        header_key: Authorization
-        header_value: Token b9f100590925b529664ed9d370f5f8da124b2c20
-
-
-Tokens configuration
---------------------
-Create or update the (single) YAML configuration file with your settings:
-
-.. code-block:: yaml
+Objecttypes API
+===============
 
-    tokenauth_config_enable: true
-    tokenauth:
-      items:
-        - identifier: token-1
-          token: ba9d233e95e04c4a8a661a27daffe7c9bd019067
-          contact_person: Person 1
-          email: person-1@example.com
-          organization: Organization XYZ # optional
-          application: Application XYZ # optional
-          administration: Administration XYZ # optional
-          permissions:
-            - object_type: b427ef84-189d-43aa-9efd-7bb2c459e281
-              mode: read_and_write
-
-        - identifier: token-2
-          token: 7b2b212d9f16d171a70a1d927cdcfbd5ca7a4799
-          contact_person: Person 2
-          email: person-2@example.com
-          permissions:
-            - object_type: b0e8553f-8b1a-4d55-ab90-6d02f1bcf2c2
-              mode: read_only
-              use_fields: true
-              fields:
-                '1':
-                  - record__data__leeftijd
-                  - record__data__kiemjaar
-
-.. note:: To ensure the proper functioning of the tokens, it is essential to first configure the ``objecttypes``.
-          Then, the token configuration must be completed to guarantee the correct configuration of the ``Permissions``.
-
-
-Mozilla-django-oidc-db
-----------------------
-
-Create or update the (single) YAML configuration file with your settings:
+.. FIXME these can currently only be included manually, because these docs are built in
```

Comment:
Not sure if there is a better way to do this, unless we want to give Objecttypes its own RTD page